tests/MongoDB.Driver.Legacy.Tests/MongoCollectionTests.cs

```diff
@@ -3491,22 +3401,19 @@ public void TestValidateWithFull()
         [Fact]
         public void TestValidateWithMaxTime()
         {
-            if (_primary.Supports(FeatureId.MaxTime) && _primary.InstanceType != MongoServerInstanceType.ShardRouter)
```

Comment:
Keep the second part of the test (the shard router part)

Reply:
Done.